### ISSUE TITLE
Missing Textures Issue in the latest update. 

### ISSUE THREAD

**Issue by TheGodElf** [Y0ungSandwich/Mech-and-Magic#191]
So in the recent update to Mech and Magic (v 2.0.6) I found that the Textures for the Secret Rooms Mod were missing. I had ghost blocks in my base as a secret entrance and when i downloaded the update they just showed up as missing. I went back to a previous base that I had and even Secret Trapdoors were missing textures too. I told KamizuraSan and he told me to check in Single player. Which i did and the same result happened. He told me to post it here and tell you about it.

**Comment by Y0ungSandwich** [comment]
Im seeing all textures here. Try deleting all mods from ATL folders and a fresh install of the pack. The mod may have gotten corrupted on install.

**Comment by TheGodElf** [comment]
I'll check when i get back onto my computer but i took a screen shot to show that they were coming up missing for me. In a single player world that is.
![imgpsh_fullsize](https://user-images.githubusercontent.com/36866658/36705388-e511ae50-1b32-11e8-88d2-49f6a69d3b79.png)

**Comment by Y0ungSandwich** [comment]
No pics needed I fully believed you. Files getting derped on download is nothing new.

**Comment by TheGodElf** [comment]
Anyway realized i posted this on the forum for the wrong version of the modpack XD but i tried what you suggested. Deleted the entire pack from my Instances folder and reinstalled it but the result was the same

**Comment by Y0ungSandwich** [comment]
![2018-02-27_16 35 31](https://user-images.githubusercontent.com/3539387/36756183-49033420-1bdc-11e8-802e-199ebead4a0b.png)
Very odd indeed.  Ill see what I can dig up.

**Comment by TheGodElf** [comment]
Idk what it could be then. Really have no idea. I mean it's not the end of the world. My base is claimed so if anyone else sees it they can't get to it. It's just visually bothering a little. I could live with it worst case scenario.

**Comment by TheGodElf** [comment]
On another note my At Launcher console is giving me a bunch of Texture Errors when i load the pack

**Comment by Y0ungSandwich** [comment]
Yeah that just screams messed up download to me. I'd try a reinstall but first change the server in the launcher settings.

**Comment by TheGodElf** [comment]
I'll give it a try.

**Comment by Y0ungSandwich** [comment]
Anything new on this?

**Comment by TheGodElf** [comment]
Nothing. I tried deleting all the data and reinstalling it on multiple servers (you know changing the download server) And it's still the same for me. I'm still playing the game so i guess if it can't be fixed it can't be helped. Maybe it's something with my computer.
